### ISSUE TITLE
13.4.19 breaks 'use client' functionality

### ISSUE THREAD

**Issue by hencca** [vercel/next.js#54913]
### Link to the code that reproduces this issue or a replay of the bug

https://github.com/hencca/next-js-client

### To Reproduce

1. create new project npx create-next-app@lates
2. currently will install 13.4.19
3. make a simple client component that has one button and alerts on click

`"use client"
import React from 'react';

const MyComponent = () => {
    return (
        <button onClick={()=>alert("Hi")}>
            Say Hi
        </button>
    );
};

export default MyComponent;`

4. after npm run dev, click silently fails

### Current vs. Expected behavior

Expected: button should trigger alert.
Current: doesn't

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
Operating System:
      Platform: darwin
      Arch: x64
      Version: Darwin Kernel Version 22.6.0: Wed Jul  5 22:21:56 PDT 2023; root:xnu-8796.141.3~6/RELEASE_X86_64
    Binaries:
      Node: 16.13.1
      npm: 9.7.2
      Yarn: 1.22.19
      pnpm: N/A
    Relevant Packages:
      next: 13.4.20-canary.15
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
      typescript: 5.2.2
    Next.js Config:
      output: N/A
```

### Which area(s) are affected? (Select all that apply)

Not sure

### Additional context

branch **main** doesnt work
branch **works** works

**Comment by znonthedev** [comment]
Duplicate of https://github.com/vercel/next.js/issues/54269

TLDR; Upgrade Node.js

**Comment by hencca** [comment]
alright. closing
